src/js/stores/AppsStore.js

```diff
@@ -85,18 +85,16 @@ function getAppHealth(app) {
     healthData[i].quantity = Math.min(capacityLeft, healthData[i].quantity);
   }
 
-  // ... show everything above that in blue
+  // get the number for tasks overcapacity
```

Comment:
I have to say these comments look pretty useless to me. Kinda like:

```
// increment counter by 1
counter++;
```
:laughing: 

Let's just kick them out.